.github/workflows/build-shopware-image.yml

```diff
@@ -0,0 +1,49 @@
+name: Build Shopware Image
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  build-test-image:
```

Comment:
we should build the image for the newest release in 6.6.x and trunk like before in addition to 6.6.x and trunk. You can use the action [shopware/github-actions/versions@main](https://github.com/shopware/shopware/blob/e3c4e85f881e18141bfb87a9ff261afb33e6e5a0/.github/workflows/integration.yml#L330-L352)

The ats should only test against released versions not trunk (because it should continue to work with older versions, otherwise it's pretty useless for externals)

Reply:
That makes sense; I'll add builds for more versions and remove the test runs for trunk then.